src/config.rs

```diff
@@ -100,6 +101,10 @@ impl Config {
         Ok(Self::get_folder()?.join(GLOBAL_CONFIG_FILE_NAME))
     }
 
+    pub fn get_wax_file_path() -> Result<PathBuf, GlobalConfigError> {
+        Config::get_folder().map(|config_folder| config_folder.join(GLOBAL_WAX_INDEX_FILE_NAME))
+    }
```

Comment:
After offline discussion:

We should use this index file to store a mapping between commands and it's package names (not the full path where they are installed).

Eg.
```
{
  "python": "python@0.1.0"
}
```